### ISSUE TITLE
Tell contributors they need to validate HTML when testing

### ISSUE THREAD

**Issue by EoinShaughnessy** [alphagov/govuk-frontend#2279]
## What

Add step to [our Frontend testing doc](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/testing.md) about checking that HTML is valid.

## Why

While working on [accessibility criteria content](https://github.com/alphagov/govuk-design-system/issues/1792), we realised that our testing doc lacked content about checking markup (also, no link to the validator site). That could mean that contributors do not know they will need to check their HTML when they test their contribution.

## Who needs to know about this

@hannalaakso, @EoinShaughnessy  

## Done when

- [x] Update added to doc
- [ ] Update PR reviewed and approved by dev
- [ ] Update 2i'd
